### DISCUSSION TITLE
More Monetization Methods ❤️

### DISCUSSION THREAD

**Discussion by yunus-cakir** [modrinth/.github#154] [Feature requests]
Hi! Could you add another monetization method please? I know Paypal is a universal and trusty method but they are not in Turkey.
If you don't have any thoughts what to add, I can suggest Papara btw ❤️
![image](https://github.com/modrinth/.github/assets/83448525/dd8027fe-87da-4e38-91b6-b541eac01939)

**Comment by Madis0** [comment]
I suggest [Wise](https://wise.com/gb/business/), [API docs here](https://wise.com/gb/business/api).

**Comment by triphora** [comment]
We have added gift card and US-based ACH payouts.

**Reply by yunus-cakir** [reply]
Hi thanks for the thought but could you add Binance, BTC or something like crypto please? I tried almost every option but they are not workin' in Turkey even Amazon.com. (We have .com.tr)
